addon/components/o-s-s/button.ts

```diff
@@ -73,6 +74,8 @@ interface ButtonArgs {
   skin?: string;
   size?: string;
   loading?: boolean;
+  showLabelOnLoad?: boolean;
```

Comment:
Not used, no?

Reply:
Yep! it was my initial solution. Removed.